source/tasks/OctoInstaller/OctoInstallerV5/octopusCLIVersionFetcher.ts

```diff
@@ -0,0 +1,67 @@
+import { maxSatisfying, valid } from "semver";
+
+export class OctopusCLIVersionFetcher {
+    constructor(readonly versions: string[]) {}
+
+    public getVersion(versionSpec: string) {
+        if (versionSpec === "*") {
```

Comment:
I think we should make "latest" a synonym for "*"

Reply:
I understand we did called it `latest` before, but not sure we should

Reply:
The UX here is my concern. If I have a step with v4 selected and then switch to v5, my expectation would be that `latest` should either be rejected by the save or should be accepted to mean what it previously meant. Yaml can't support the idea of stopping the update based on validation, which is why I was leaning towards continuing to support it.
I couldn't get our test pipeline to run, what does the user see if `latest` is used?

Reply:
They will see an error, see https://github.com/OctopusDeploy/OctoTFS/pull/230/files#diff-61116d68378cf54c8ea8621ba761b95a9b1340f0cff243c15504b600f1501968R20

Reply:
fair enough.